tests/manage/test_pvresize_with_node_down.py

```diff
@@ -0,0 +1,228 @@
+OUR_PVC = None
+POD = None
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Finalize teardown and call setup
+    """
+    def finalizer():
+        teardown()
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def check_obj(obj_type):
+    """
+    Check pvc and pod.  Only one of each will be created in this test.
+
+    Return metadata
+    """
+    cmd_str = f'oc -n {config.ENV_DATA["my_namespace"]} get {obj_type} -o json'
+    objs = json.loads(run_cmd(cmd_str))
+    assert len(objs['items']) == 1
+    log.info(f"{obj_type} created successfully.")
+    return objs['items'][0]['metadata']
+
+
+def setup():
+    """
+    Setting up the environment for the test
+
+    Create namespace
+    Create storageclass
+    Create pvc
+    Create pod
+    """
+    NAMESPACE.create(resource_name=OCS_BUG_ID)
+    config.ENV_DATA["my_namespace"] = OCS_BUG_ID
+
+    pvc_data = templating.load_yaml_to_dict(constants.CSI_PVC_YAML)
+    pvc_data['metadata']['namespace'] = config.ENV_DATA["my_namespace"]
+    pvc_data['spec']['storageClassName'] = 'rook-ceph-block'
+    pvc_data['spec']['resources']['requests']['storage'] = '100Gi'
+    pvc_name = pvc_data['metadata']['name']
+    global OUR_PVC
+    OUR_PVC = PVC(**pvc_data)
+    OUR_PVC.create()
+    pvc_info = check_obj('pvc')['annotations']
+    while 'pv.kubernetes.io/bind-completed' not in pvc_info:
+        sleep(1)
+        pvc_info = check_obj('pvc')['annotations']
+    assert pvc_info['pv.kubernetes.io/bind-completed'] == 'yes'
+    assert pvc_info['pv.kubernetes.io/bound-by-controller'] == 'yes'
+
+    pod_data = templating.load_yaml_to_dict(constants.CSI_RBD_POD_YAML)
+    pod_data['metadata']['namespace'] = config.ENV_DATA["my_namespace"]
+    pod_data['metadata']['name'] = f'{OCS_BUG_ID}-pod'
+    first_claim = pod_data['spec']['volumes'][0]
+    first_claim['persistentVolumeClaim']['claimName'] = pvc_name
+    first_container = pod_data['spec']['containers'][0]
+    first_volume_mount = first_container['volumeMounts'][0]
+    first_volume_mount['mountPath'] = f'/mnt/{OCS_BUG_ID}'
+
+    global POD
+    POD = Pod(**pod_data)
+    POD.create()
+    pod_info = check_obj('pod')
+    assert pod_info['namespace'] == OCS_BUG_ID
+    global CEPH_TOOL
+    CEPH_TOOL = get_ceph_tools_pod()
+
+
+def teardown():
+    """
+    Cleanup
+
+    Delete pod
+    Delete pvc
+    Delete namespace
+    Delete pv
+    """
+    pv_name = OUR_PVC.backed_pv
+    POD.delete()
+    POD.delete_temp_yaml_file()
+    OUR_PVC.delete()
+    OUR_PVC.delete_temp_yaml_file()
+    NAMESPACE.delete(resource_name=OCS_BUG_ID)
+    run_cmd(f'oc delete pv {pv_name}')
+
+
+def cmd_on_my_namespace(real_cmd):
+    """
+    Run an oc command on app pod in the test namespace
+
+    Retries if the command fails.
+
+    Raises:
+        CommandFailed if a failure limit is reached
+
+    Args:
+        real_cmd (str): oc command to be run ('get pods', for example)
+    """
+    FAIL_LIMIT = 100
```

Comment:
lowercase variable name. save uppercase for global vars.

Reply:
Done